examples/redis/v1beta3/README.md

```diff
@@ -0,0 +1,118 @@
+## Reliable, Scalable Redis on Kubernetes
+
+The following document describes the deployment of a reliable, multi-node Redis on Kubernetes.  It deploys a master with replicated slaves, as well as replicated redis sentinels which are use for health checking and failover.
+
+### Prerequisites
+This example assumes that you have a Kubernetes cluster installed and running, and that you have installed the ```kubectl``` command line tool somewhere in your path.  Please see the [getting started](https://github.com/GoogleCloudPlatform/kubernetes/tree/master/docs/getting-started-guides) for installation instructions for your platform.
+
+The v1beta3 API is not enabled by default. The kube-apiserver process on master needs to be restarted with the --runtime_config=api/v1beta3 argument.
+### A note for the impatient
+This is a somewhat long tutorial.  If you want to jump straight to the "do it now" commands, please see the [tl; dr](#tl-dr) at the end.
+
+### Turning up an initial master/sentinel pod.
+is a [_Pod_](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/pods.md).  A Pod is one or more containers that _must_ be scheduled onto the same host.  All containers in a pod share a network namespace, and may optionally share mounted volumes.
+
+We will used the shared network namespace to bootstrap our Redis cluster.  In particular, the very first sentinel needs to know how to find the master (subsequent sentinels just ask the first sentinel).  Because all containers in a Pod share a network namespace, the sentinel can simply look at ```$(hostname -i):6379```.
+
+Here is the config for the initial master and sentinel pod:
+[redis-master](redis-master.yaml)
+
+Create this master as follows:
+```sh
+kubectl create -f examples/redis/v1beta3/redis-master.yaml
```

Comment:
Please ensure that all these commands are creating replication controllers, not raw pods.